Добавление спрайтов в руках для всех пда.

Blundir opened the pull request:
-Добавление спрайтов в руках для всех пда.

-Фикс неправильного отображения пда детектива.

-Добавление недостающих карт для отображения в кошельке.

По предложке: https://discord.com/channels/617003227182792704/755125334097133628/972586869583933480
![dreammaker_24hDjQOEwp](https://user-images.githubusercontent.com/100090741/195478724-31d83e2c-7f83-43d1-bd13-04b9b07d22ec.png)
![dreammaker_tzOFnbYIlW](https://user-images.githubusercontent.com/100090741/195478734-29f3271e-3b8c-46d1-b44a-582a5f7f3091.png)

Blundir committed:
Добавление спрайтов в руках для всех пда.

Добавление спрайтов в руках для всех пда.
Фикс неправильного отображения пда детектива.
Добавление недостающих карт для отображения в кошельке.

Blundir committed:
Дофикс спрайтов пда

Dimach commented:
Этот ПР будет конфликтовать с моим ПОм #1441, сможешь поребейзить или лучше мне?

Blundir commented:
> 

Лучше тебе, у меня щас около 8 пров будет и так.

Blundir committed:
Дофикс спрайтов карт в руках.

Исправление неправильного спрайта и неправильного отображения нескольких карт в руках.

Blundir committed:
Второй дофикс спрайтов карт.
